website/docs/docs/dbt-cloud-apis/sl-manifest.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Semantic manifest"
+id: sl-manifest
+description: "Learn about the semantic manifest.json file and how you can use artifacts to gain insights about your Semantic Layer."
+tags: [Semantic Layer, APIs]
+sidebar_label: "Semantic manifest"
+---
+
+<VersionBlock lastVersion="1.5">
+
+import LegacyInfo from '/snippets/_legacy-sl-callout.md';
+
+<LegacyInfo />
+
+</VersionBlock>
+
+dbt creates an [artifact](/reference/artifacts/dbt-artifacts) file called the "Semantic Manifest" (`semantic_manifest.json`), which MetricFlow requires to build and run metric queries properly for the Semantic Layer. This artifact contains comprehensive information about your Semantic Layer. It is an internal file, that acts as the integration point with MetricFlow. 
```

Comment:
is it "dbt Semantic Layer" or "Semantic Layer"? i thought it was the former but unsure 🤷‍♀️ 

it's written as a proper noun (with caps), implying it's a product name, so suggest "dbt Semantic Layer". for example, "your dbt Cloud account" versus "your Cloud account" 

```suggestion
dbt creates an [artifact](/reference/artifacts/dbt-artifacts) file called the _Semantic Manifest_ (`semantic_manifest.json`), which MetricFlow requires to build and run metric queries properly for the dbt Semantic Layer. This artifact contains comprehensive information about your dbt Semantic Layer. It is an internal file that acts as the integration point with MetricFlow. 
```